src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -121,6 +121,13 @@ public class AgentKnobs
             new EnvironmentKnobSource("VSTSAGENT_TRACE"),
             new BuiltInDefaultKnobSource(string.Empty));
 
+        public static readonly Knob DumpJobEventLogs = new Knob(
+            nameof(DumpJobEventLogs),
+            "If set to anything, dump event viewer logs",
+            new RuntimeKnobSource("VSTSAGENT_DUMP_JOB_EVENT_LOG"),
+            new EnvironmentKnobSource("VSTSAGENT_DUMP_JOB_EVENT_LOGS"),
+            new BuiltInDefaultKnobSource(string.Empty));
```

Comment:
Could it make sense to make it 'false' - to align with other similar knobs? 
Currently VSTSAGENT_DUMP_JOB_EVENT_LOG=false will also enable logs dump instead of disabling, which could confuse

Reply:
Makes sense. Fixed!